src/utils/contexts/balances/hook.tsx

```diff
@@ -0,0 +1,44 @@
+import { CurrencyExt } from '@interlay/interbtc-api';
```

Comment:
I'm not sure about the location of this hook in this folder, I would keep hook in utils/hooks and context in utils/contexts, what do you think?

Reply:
moved.